constellations/gemini-user/src/account/account.js

```diff
@@ -0,0 +1,95 @@
+"use strict";
+require("./account.sass");
+import React from "react";
+import API from "../utils/api.js";
+import { union } from "lodash";
+
+import { SettingsTab } from "./settings.js";
+import {
+    RegistrationsTab,
+    RegistrationsTabMain,
+    RegistrationsTabAllClasses,
+} from "./registrations.js";
+import { PaymentTab } from "./payment.js";
+
+export class AccountPage extends React.Component {
+    state = {
+        id: 0,
+        selectedTab: "settings",
```

Comment:
when we use constants, it's best to define constant variables at the top of the file.

So at the top I would have:
```
const TAB_SETTINGS = "settings";
const TAB_REGISTRATIONS = "registrations";
const TAB_PAYMENT = "payment";
```

then here use: `selectedTab: TAB_SETTINGS`

It's better to do things this way because you force tab selection to use "typed" variables instead of a value where you are prone to typos.
For example, if you had `selectedTab: "setings"`, you might not notice this problem until your tab selection doesn't work. However, if you did `selectedTab: TAB_SETINGS` you would get a compile error and immediately alert you!